float64 setpoint_data
float64 measured_data
